### ISSUE TITLE
Invitation problem

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Neseek77/openmeetings#534]
```
Which version of OpenMeetings are you running?
0.5.2 r1593

What's your operating system on client and server side?
Windows client, linux as server, tried firefox, opera and IE on the client

Is it a problem related to the client/usability or server-side?
I think it`a a server problem

Did you have any trace/debug output?
ERROR 09-27 03:57:54.176  MailHandler.java 2703511 184 
org.openmeetings.utils.mail.MailHandler [pool-3-thread-3] - [mail send] 
java.lang.NumberFormatException: For input string: ""


To get the debug output of the client run the software in debug-modus
Debug modus of the client:
http://$RED5-HOME:$RED5-HTTPPORT/xmlcrm/maindebug.lzx.lzr=swf8.swf

netRemoteCallHib/doCall: [  invitationservice.sendInvitationHash  ] 
«lz.inviteMainWindow#120» 
«Date#121| Sat Sep 27 03:00:00 GMT+0300 2008» 
«Date#122| Sun Sep 28 03:00:00 GMT+0300 2008» 
sendInvitationHash :  Errorjava.lang.NumberFormatException: For input 
string: "" 
explorerBox/setTintRGB «lz.view#123» 
explorerBox/setTintRGB «Object#124| {ra: 176, ga: 196, ba: 222, rb: 0, gb: 
0, bb: 0}» 



so basically, I click on "Send Invitation", I complete an email address, a 
message, a password, and when I submit it, a window appears with the 
error: Errorjava.lang.NumberFormatException: For input string: "" 

```

Original issue reported on code.google.com by `theol...@gmail.com` on 26 Sep 2008 at 11:09
